pipeline/test_transform.py

```diff
@@ -1,4 +1,136 @@
-# pylint: skip-file
 """Test file for the transform pipeline."""
 
+# pylint: skip-file
 import pytest
+from unittest.mock import patch
+import pandas as pd
+
+from transform import (get_required_columns,
+                       check_valid_sale,
+                       clean_urls,
+                       handle_missing_values,
+                       validate_foreign_keys,
+                       get_id_mapping,
+                       filter_by_currency,
+                       add_foreign_key_column,
+                       clean_dataframe,
+                       export_dataframe)
+
+pytest.skip(allow_module_level=True)
+
+
+def test_get_required_columns_returns_the_required_columns(sample_df):
+    result = get_required_columns(sample_df)
+    expected_cols = [
+        "utc_date", "artist_name", "item_type", "item_description", "album_title",
+        "currency", "amount_paid", "country",
+        "url", "art_url"
+    ]
+    assert list(result.columns) == expected_cols
+    assert result.shape[1] == len(expected_cols)
+
+
+def test_get_required_columns_raises_error(empty_df):
```

Comment:
I'd change this test name to be more lengthy, while the one Xac commented on above repeats itself, this one doesn't really tell me what it does exactly if I see it fail on the pytest terminal. How about

```suggestion
def test_get_required_columns_errors_when_dataframe_empty(empty_df):
```